lms/djangoapps/django_comment_client/base/tests.py

```diff
@@ -851,8 +852,12 @@ def setUp(self):
         self.student = UserFactory.create()
         CourseEnrollmentFactory(user=self.student, course_id=self.course.id)
 
+    @patch('lms.lib.comment_client.comment.Comment.thread')
```

Comment:
@jimabramson In this case, apparently not. The code this case was once testing had a side effect that required some data from the parent thread, which we did not need, and which would not exist in this case. Removing it has no adverse effects, so I will do so.